docs/fundamentals/flow/monitoring-flow.md

```diff
@@ -0,0 +1,144 @@
+(monitoring-flow)=
+# Monitor Flow
+ 
+A Jina {ref}`Flow <flow-cookbook>` exposes several core metrics that allow you to have a deeper look
+at what is happening inside it. Metrics allow you to, for example, monitor the overall performance 
+of your Flow, detect bottlenecks, or alert your team when some component of your Flow is down.
+
+Jina Flows expose metrics in the [Prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/). This is a plain text format that is understandable by both human and machine. These metrics are intended to be scraped by
+[Prometheus](https://prometheus.io/),an industry standard tool for collecting and monitoring metrics.
+
+To visualize your metrics through a dashboard, we recommend [Grafana](https://grafana.com/)
+
+
+## Using the monitoring in a Flow
```

Comment:
This heading and the one above say basically the same thing. What exactly is the purpose of this sub-section vs the one above?

Reply:
`Monitor Flow` and `Using the monitoring in a Flow`. Should the second one be `Monitor the entire Flow`? That would make sense imo, but then further down below we should also have `Monitor select Pods in a Flow` or something like that